### ISSUE TITLE
Static method calls in array literals are broken by formatter

### ISSUE THREAD

**Issue by sbleon** [jcberquist/commandbox-cfformat#129]
Input:
```
component {

  bleh = [bar::foo()];

}
```

Output:
```
component {

  bleh = [bar: : foo()];

}
```

The extra spaces between and after the colons are bad.

Workaround: surround static call with parens:
```
component {

  bleh = [(bar::foo())];

}
```

This may be a problem with the underlying Sublime syntax file, as I've noticed the syntax highlighting changes when I add the parens:
![Screen Shot 2021-09-21 at 2 16 58 PM](https://user-images.githubusercontent.com/198711/134225629-01c519ff-05b3-44aa-8fba-f81327b459b0.png)
vs
![Screen Shot 2021-09-21 at 2 17 32 PM](https://user-images.githubusercontent.com/198711/134225693-9edfaaf0-0112-477c-b69d-4bdba78aeaf0.png)

This may be related to #128.

**Comment by jcberquist** [comment]
Thanks for the issues, this and #128 should be fixed in `v0.16.7`.

**Comment by sbleon** [comment]
Thanks for the most responsive maintainer ever! You're the best!
